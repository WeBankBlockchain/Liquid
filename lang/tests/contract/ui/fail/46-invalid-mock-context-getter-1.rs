use liquid_lang as liquid;

#[liquid::interface(name = auto)]
mod foo {
    extern "liquid" {
        #[liquid(mock_context_getter = something)]
        fn foo(&self);
    }
}

fn main() {}