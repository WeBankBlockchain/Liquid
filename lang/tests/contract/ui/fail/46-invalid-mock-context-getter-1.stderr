error: the attribute `mock_context_getter` should be assigned with a literal string
 --> $DIR/46-invalid-mock-context-getter-1.rs:6:17
  |
6 |         #[liquid(mock_context_getter = something)]
  |                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
